src/Components/List.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+
+
+class List extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state ={
+            viewList: (item) => {
+                return item
+            }
+        }
+    }
+
+    render(){
+    return(
+        <section className="main">
+            <input id="toggle-all" className="toggle-all" type="checkbox" />
+            <label htmlFor="toggle-all">Mark all as complete</label>
+            <ul className="todo-list">
+                {
+
+                    this.props.todoList.filter(this.state.viewList).map((item) => {
+                    return (
+                        <li className={item.chacked? "completed" : ''} >
+                            <div className="view">
+                                <input
+                                    className="toggle"
+                                    type="checkbox"
+                                    checked={item.chacked} onClick={() => {
+                                        this.props.changeStateItem(item);
+
+                                    }}
+                                />
+
+                                <label>{item.name}</label>
+
+                                <button className="destroy" onClick={() => {
+                                    this.props.deleteItem(item)
+                                }}>
+                                </button>
+
+                            </div>
+                        </li>
+                        )}
+                    )}
+            </ul>
+            <footer className="footer">
+                <span className="todo-count">
+                    {
+                        this.props.todoList.reduce((accum, item ) => {
```

Comment:
Please install eslint and format your code properly. 
It's quite hard to read unformatted code.
```suggestion
  this.props.todoList.reduce(( accum, item ) => {
    if (!item.checked) {
      return accum + 1;
    } else {
      return accum;
    }
  }, 0)
```
Suggestion 2: you might not need `reduce`. You can easily use `filter`.
```javascript
{this.props.todoList.filter((item) => !item.checked).length}
```